jetty-ee11/jetty-ee11-servlet/src/test/java/org/eclipse/jetty/ee11/servlet/ServletContextResourcesTest.java

```diff
@@ -77,7 +77,7 @@ public void init() throws Exception
         connector = new LocalConnector(server);
         connector.getConnectionFactory(HttpConfiguration.ConnectionFactory.class).getHttpConfiguration().setSendServerVersion(false);
 
-        Path resBase = MavenTestingUtils.getTestResourcePathDir("contextResources");
+        Path resBase = TestEeResources.getResourceAsPathDir("/contextResources");
```

Comment:
Since TestEeResources houses all of these directories, files, jars, perhaps there should be a few constants we can use here.

```suggestion
        Path resBase = TestEeResources.getResourceAsPathDir(TestEeResources.CONTEXT_RESOURCES);
```